packages/primitives/background/package.json

```diff
@@ -14,11 +14,11 @@
   "sideEffects": false,
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "@primitives/block": "^0.5.0",
-    "colorido": "^0.1.0"
+    "@primitives/block": "^0.5.0"
   },
   "devDependencies": {
     "@types/react": "^16.8.6",
+    "colorido": "^0.1.0",
```

Comment:
where do we use it?

Reply:
I was writing a long response only to realize that you are right. Yep. I'll fix that